core/src/main/java/io/aiven/klaw/service/UtilControllerService.java

```diff
@@ -611,12 +614,27 @@ public AuthenticationInfo getAuth() {
       authenticationInfo.setAddDeleteEditEnvs(addDeleteEditEnvs);
 
       // coral attributes
-      authenticationInfo.setCoralEnabled(Boolean.toString(coralEnabled));
+      authenticationInfo.setCoralEnabled(Boolean.toString(coralEnabled && isCoralBuilt()));
 
       return authenticationInfo;
     } else return null;
   }
 
+  private boolean isCoralBuilt() {
+
+    if (isCoralBuilt == null) {
+      ClassPathResource file =
+          new ClassPathResource(
+              "classpath:templates/coral/index.html", this.getClass().getClassLoader());
+      if (file != null) {
```

Comment:
this condition always seems to be true

Reply:
I had put it in as a safety check, 
Looking at the code for ClassPathResource it shouldn't technically ever return null so I think I can remove that check.